backend/src/server/plugins/api-metrics.ts

```diff
@@ -0,0 +1,21 @@
+import opentelemetry from "@opentelemetry/api";
+import fp from "fastify-plugin";
+
+export const apiMetrics = fp(async (fastify) => {
+  const apiMeter = opentelemetry.metrics.getMeter("API");
+  const latencyHistogram = apiMeter.createHistogram("API latency", {
```

Comment:
Curious: what is the impact of such metric collection on the API? Does it increase our API response times much?